tests/test_generate_lora.py

```diff
@@ -0,0 +1,110 @@
+import json
+import subprocess
+import sys
+from contextlib import redirect_stdout, redirect_stderr
+from io import StringIO
+from pathlib import Path
+from unittest import mock
+from unittest.mock import Mock, call, ANY
+
+import pytest
+import torch
+
+
+@pytest.mark.parametrize("max_seq_length", (10, 5 + 20))
+def test_generate(max_seq_length):
+    import generate.lora as generate
```

Comment:
generate.lora is exactly generate.base. Making this test redundant. There's no lora-specific generation code

Reply:
Damn, I wasn't attentive enough. Sorry for that.
Indeed `generate.lora` uses `generate.base:generate`.